build-wasm.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
```

Comment:
maybe we should organize all this inside one make file which already exists inside docker folder to reuse some steps?

Reply:
+1

Reply:
I'll add a separate issue about restoring local Docker-based builds